.doc_gen/metadata/ssm_metadata.yaml

```diff
@@ -31,3 +49,54 @@ ssm_PutParameter:
                 - ssm.rust.create-parameter
   services:
     ssm: {PutParameter}
+ssm_CreateOpsItem:
+  title: Create a new OpsItem using an &AWS; SDK
+  title_abbrev: Create a new OpsItem
+  synopsis: create a new OpsItem.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ssm
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - ssm.java2.create_ops.main
+  services:
+    ssm: {CreateOpsItem} 
+ssm_DescribeOpsItems:
+  title: Describe an OpsItem using an &AWS; SDK
+  title_abbrev: Describe an OpsItem
+  synopsis: describe an OpsItem.
+  category:
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ssm
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - ssm.java2.describe_ops.main
+  services:
+    ssm: {DescribeOpsItems}
+ssm_UpdateOpsItem:
+  title: Describe an OpsItem using an &AWS; SDK
```

Comment:
This has the wrong title (same as previous) and does not show up in the code library.